var/spack/repos/builtin/packages/openkim-models-v2/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class OpenkimModelsV2(CMakePackage):
+    """OpenKIM is an online framework for making molecular simulations
+       reliable, reproducible, and portable. Computer implementations of
```

Comment:
Would someone install `openkim-models-v2` without installing `kim-api-v2`?

This package docstring is the same as for `kim-api-v2` so perhaps the usefulness of a separate package would be clearer if explained here. I'm not familiar with these packages: perhaps it is obvious to anyone who would use them.

Also note that some packages (e.g. `llvm`) have extra modules that are controlled as variants and installed as part of that package. This approach isn't required but I thought it might be worth a mention.

Reply:
In other packaging systems the openkim-models-v2 package would have a build and run-time dependency on the kim-api-v2 package.  However, the openkim-models-v2 package files *must* be installed in the same prefix as the kim-api-v2 package in order for everything to work correctly.  So it seemed like the "extension" mechanism of spack was the right way to go.

The kim-api-v2 package is operational and useful without openkim-models-v2, but not vice-versa.

Reply:
Versions of openkim-models-v2 will kim-api-v2 will be released on different schedules which makes me feel that these should be separate packages.  Not all users would want the openkim-models-v2 package.